### PR TITLE
Added categories to movies

### LINKED ISSUE #83: Organizing movies in categories
Since we already have a lot of movies added to this list, maybe we should start organizing them in different categories like we already categorized books?

### PR DESCRIPTION
**Pull request by RichardLitt**
A first stab at this. This should close #83, but we may have to do a bit of editing afterwards.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Damn, seems like on of your other PRs conflicted with this one. Mind fixing the merge conflict?

File: readme.md [hunk 1]
```diff
@@ -802,6 +802,7 @@ I think of these stories often; The Death of Dr. Island won a Nebula and offers
 #### [2001: A Space Odyssey](http://www.imdb.com/title/tt0062622/) (1968) [8.3]
 
 Director: _Stanley Kubrick_
```
Comment (sindresorhus) [inline readme.md]: You need to add two blank spaces on the end of this to force a linebreak. Lol Markdown.

This applies to everywhere `Categories` is together with `Director`.

**Comment by RichardLitt** [comment]
Done!

**Comment by sindresorhus** [comment]
Excellent :)

**Pull request merged**